example/src/app/ng-reflect-as-text/ng-reflect-as-text.component.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NgReflectAsTextComponent } from './ng-reflect-as-text.component';
+
+describe('NgReflectAsTextComponent', () => {
+  let component: NgReflectAsTextComponent;
+  let fixture: ComponentFixture<NgReflectAsTextComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ NgReflectAsTextComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(NgReflectAsTextComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
I think this test is obsolete. We have it in many other tests already. This will save us some time.

Reply:
Done